addons/project/static/src/js/right_panel/project_right_panel.js

```diff
@@ -34,19 +34,22 @@ export default class ProjectRightPanel extends LegacyComponent {
     }
 
     formatMonetary(value, options = {}) {
-        const valueFormatted = formatMonetary(value, {
-            currencyId: this.state.data.currency_id,
+
+        const valueFormattedToInteger = formatMonetary(value, {
             ...options,
+            'digits': [false, 0],
             'noSymbol': true,
         });
+
         const currency = session.currencies[this.state.data.currency_id];
-        if (!currency) {
-            return valueFormatted;
+         if (!currency) {
+            return valueFormattedToInteger;
         }
+
         if (currency.position === "after") {
-            return `${valueFormatted}\u00A0${currency.symbol}`;
+            return `${valueFormattedToInteger}\u00A0${currency.symbol}`;
         } else {
-            return `${currency.symbol}\u00A0${valueFormatted}`;
+            return `${currency.symbol}\u00A0${valueFormattedToInteger}`;
```

Comment:
Couldn't we just use `formatFloat` here?
I feel like there is no point in using `formatMonetary` without a currency and to get a result without symbol